sim/warrior/arms/talents.go

```diff
@@ -59,6 +60,7 @@ func (war *ArmsWarrior) applyTasteForBlood() {
 		ClassSpellMask: warrior.SpellMaskRend,
 		ICD:            5 * time.Second,
 		ProcChance:     procChance,
+		SpellFlags:     core.SpellFlagAPL,
```

Comment:
You can't have triggers as APL spells. APL spells are spells that the agent can cast from the APL Cast action

Reply:
Removed.  I was trying to make the proc ICD visible.